src/test/regress/output/appendonly.source

```diff
@@ -1776,3 +1776,44 @@ LINE 1: explain select cmax, * from test_reporting_r;
 drop table test_reporting;
 drop table test_reporting_r;
 drop table test_reporting_c;
+-- Test AO functions with execution location on dispatcher.
+drop table if exists t1;
+drop table if exists t2;
+drop table if exists d;
```

Comment:
If you use "if exists", it is better to add this lines to the "-- start_ignore" - "-- end_ignore" block.

Reply:
fixed